apps/kyc-server/src/__tests__/kyc-reviews.test.ts

```
@@ -0,0 +1,170 @@
+	})
+
+	it('should handle invalid review ID', async () => {
+		const reviews = await mockDbOperations.select().from()
+		const review = reviews.find((r) => r.id === 'invalid-id')
+		expect(review).toBeUndefined()
+	})
+
+	it('should handle invalid input data', async () => {
+		const invalidData = {
+			user_id: 'test-user-123',
+			status: 'invalid-status' as any, // Force invalid status
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)